ihatemoney/tests/tests.py

```diff
@@ -1449,21 +1445,15 @@ def test_weighted_bills(self):
                               headers=self.get_auth("raclette"))
 
         expected = {
-            "active_members": [
-                {"activated": True, "id": 1, "name": "alexis", "weight": 1.0},
-                {"activated": True, "id": 2, "name": "freddy familly", "weight": 4.0},
-                {"activated": True, "id": 3, "name": "arnaud", "weight": 1.0}
+            "members": [
+                {"activated": True, "id": 1, "name": "alexis", "weight": 1.0, "balance": 20.0},
+                {"activated": True, "id": 2, "name": "freddy familly", "weight": 4.0, "balance": -20.0},
```

Comment:
To bad, this line is now too long for Flake8 to validate it. Can you split it in another line?